src/utils/env.ts

```diff
@@ -16,3 +16,14 @@ export function getVisRequestServer() {
 export function getServiceIdentifier() {
   return process.env.SERVICE_ID;
 }
+
+/**
+ * Get the list of disabled tools from environment variables.
+ */
+export function getDisabledTools(): string[] {
+  const disabledTools = process.env.DISABLED_TOOLS;
+  if (!disabledTools || disabledTools === "undefined") {
```

Comment:
I think we don't need to handle the "undefined" string additionally. If a user inputs "undefined", it means they want to disable the tool named "undefined", even though none exists.

Otherwise, following the same logic, we would also need to handle the strings "null" and "nil".